rf_tests.go

```diff
@@ -342,13 +342,20 @@ func deleteRFML(c cliContext) error {
 	return nil
 }
 
-// uploadRFML is a wrapper around test creating/updating functions
-func uploadRFML(c cliContext, api rfmlAPI) error {
+// uploadTests is a wrapper around test creating/updating functions
+func uploadRFTests(c cliContext, api rfAPI) error {
```

Comment:
Function name differs from comment?
```suggestion
func uploadTests(c cliContext, api rfAPI) error {
```

Reply:
Do you prefer `uploadtests`? I initially went with `uploadTests` and changed it to `uploadRFTests`. I think it doesn't make a huge difference here, but I think naming file `rf_tests` rather than the generic `tests` is probably better and wanted to make the function names match.

Reply:
Yeah, I wondered about this as well and wasn't sure. But thinking about it, this function is in the main package and is not exported, so there shouldn't be any clashes with any functions we might want to have in the future, and it seems unlikely we'll want to upload tests other than RF tests, so make it `uploadTests`? But it's nbd - because it's not like we can't change it in the future anyway.